spec/lib/oauth/refresh_token_request_spec.rb

```diff
@@ -67,6 +69,53 @@ module Doorkeeper::OAuth
       expect(subject).to be_valid
     end
 
+    context 'refresh tokens expire on access token use' do
+      let(:server) do
+        double :server,
+               access_token_expires_in: 2.minutes,
+               custom_access_token_expires_in: ->(_oauth_client) { 1234 },
+               refresh_token_revoked_in: 1.day
+      end
```

Comment:
Used to trigger the new behaviour.
Without it the spec `it 'does not revoke the previous token' do` will fail.